h2o-py/h2o/job.py

```diff
@@ -51,7 +51,8 @@ def poll(self):
         completion. During this time we will display (in stdout) a progress bar with % completion status.
         """
         try:
-            pb = ProgressBar(self._job_type + " progress")
+            hidden = not H2OJob.__PROGRESS_BAR__
```

Comment:
Yes, `h2o.no_progress()` to disable the progress bar, and `h2o.show_progress()` to re-enable